packages/flutter_tools/lib/src/build_system/targets/android.dart

```diff
@@ -8,82 +8,15 @@ import '../../base/file_system.dart';
 import '../../build_info.dart';
 import '../../globals.dart' as globals;
 import '../build_system.dart';
-import '../depfile.dart';
 import '../exceptions.dart';
-import 'assets.dart';
-import 'dart.dart';
-import 'icon_tree_shaker.dart';
-
-/// Prepares the asset bundle in the format expected by flutter.gradle.
-///
-/// The vm_snapshot_data, isolate_snapshot_data, and kernel_blob.bin are
-/// expected to be in the root output directory.
-///
-/// All assets and manifests are included from flutter_assets/**.
-abstract class AndroidAssetBundle extends Target {
```

Comment:
Most apps share the same bundle structure so I refactored this into a common class.